composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "laravel/framework": "^9.19",
+        "laravel/framework": "^9.0 || ^10.0 || ^11.0",
```

Comment:
I think this will restrict to these versions only so we may not support the updates of laravel like so.

I think we may make something like

"9.*|10.*|11.*"
or 
">=9"